person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] Let's add `:rental` to attr_accessor as users can write and read the rental.

Reply:
Thanks for the suggestion I'll implement it!